cmd/generatediagram.go

```diff
@@ -151,18 +156,24 @@ skinparam component {
 	ArrowFontColor #636363
     ArrowSize 10pt
 }
+skinparam interface {
+    backgroundColor<<not-found public service>> Red
+    backgroundColor<<not-found private service>> Red
+    backgroundColor<<not-found public stream>> Red
+    backgroundColor<<not-found private stream>> Red
+}
 `)
 	addValidationNote := func(id string, name string, vr *jwt.ValidationResults) {
 		if len(vr.Issues) == 0 {
 			return
 		}
-		if len(vr.Issues) == 1 && strings.HasPrefix(vr.Issues[0].Description, "no activation provided for import") {
+		if len(vr.Issues) == 1 && strings.HasPrefix(vr.Issues[0].Description, "the field to has been deprecated") {
```

Comment:
should this type of stuff be in a `nsc validate` - which then runs validations all we want on the store including all accounts?

Reply:
I am thinking that otherwise we are adding all sorts of external validations - better - maybe we should do this as `warnings` from the JWT library...

Reply:
These are warnings, just in this one context I don't want to show that one as it blows up the diagram.
I hope that no one will ever have to deal with that warnings as they use jwtv2 correctly.